js/index.js

```diff
@@ -42,6 +42,79 @@ const siteContent = {
   }
 };
 
+//---------------------------------header---------------------------------
+let links = document.querySelectorAll("header nav a");
+// links[0].textContent = siteContent["nav"]["nav-item-1"];
+// links[1].textContent = siteContent["nav"]["nav-item-2"];
+
+for (var i = 0; i < links.length; i++) {
+  links[i].textContent = siteContent["nav"][`nav-item-${i + 1}`];
+}
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+//---------------------------------cta---------------------------------
+
+let ctaH1 = (document.querySelector(".cta .cta-text h1").textContent =
+  siteContent.cta.h1);
+
+let ctaBtn = (document.querySelector(".cta-text button").textContent =
+  siteContent.cta.button);
+
+let ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute("src", siteContent["cta"]["img-src"]);
+
+//---------------------------------main-content---------------------------------
+let topContent = document.querySelector(".top-content");
+
+let botContent = document.querySelector(".bottom-content");
+
+let mC = "main-content";
+
+//top
+topContent.children[0].querySelector("h4").textContent =
+  siteContent[mC]["features-h4"];
+topContent.children[0].querySelector("p").textContent =
+  siteContent[mC]["features-content"];
+
+topContent.children[1].querySelector("h4").textContent =
+  siteContent[mC]["about-h4"];
+topContent.children[1].querySelector("p").textContent =
+  siteContent[mC]["about-content"];
+
+//middle
+let middleImg = document
+  .getElementById("middle-img")
+  .setAttribute("src", siteContent["main-content"]["middle-img-src"]);
+
+//bottom
+botContent.children[0].querySelector("h4").textContent =
```

Comment:
This was such a great way to clean up what could've been double the amount of repetitive code.
Nicely executed.